autoprof/__init__.py

```diff
@@ -4,15 +4,15 @@
 from . import utils
 from . import fit
 from . import AP_config
+import sys
+import argparse
 
```

Comment:
That is a good idea. however, import sys and argparse should be the highest level import. Going forward I would like to try and match the PEP 8 rules for imports. See also issue #23